packages/coordinator/src/services/verification/signature-verification.ts

```diff
@@ -32,6 +32,11 @@ function verifyPrescriptionSignature(parentPrescription: hl7V3.ParentPrescriptio
   return errors
 }
 
+function verifyChain(x509Certificate: crypto.X509Certificate): boolean {
+  const x509CertificateRoot = new crypto.X509Certificate(process.env.SUBCACC_CERT)
+  return x509Certificate.checkIssued(x509CertificateRoot)
+}
+
```

Comment:
As already mentioned: should we check Sub CA 1D against the Root CA, and the certificate within the prescription against the Sub CA 1D certificate? We also need to make sure we document this decision within the ticket, for future reference.

Reply:
Hi @StefanoFrazzetto, I have renamed the dummy certs in the repo as the names were causing confusion. There was one called root which wasn't a root and the trustedX509 is a mock cert i created which is only trusted by the mock subCa created it is not trusted by the INT subCa. However the tests covering the release response use prescriptions signed by an int smart card which why I have INT subCa. 

There is a comment from Tim on the ticket regarding just checking against the subCa. He is also raising a KAD to confirm the decision. 

We can catch up on this if required when  you are back.